submissions/views.py

```diff
@@ -0,0 +1,18 @@
+from django.shortcuts import render
+from hackathon.models import HackProject
+from .forms import SubmissionForm
+
+def submit(request):
+    if request.method == 'POST':  # If the form has been submitted...
+        form_data = request.POST
+        submission_form = SubmissionForm(form_data)
+        if submission_form.is_valid():
+            submission = submission_form.save()
```

Comment:
We could also consider using the **messages.success** flash notification?
"Thanks! Your project has been successfully submitted!"